changelogs/fragments/xml_example.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- xml - add an example for absent using attribute value (https://github.com/ansible-collections/community.general/pull/2644).
```

Comment:
Docs changes should not have a changelog fragment.

Reply:
Done.